tensorbay/client/profile.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+"""Statistical.
+
+:class:`Profile` is a class used to save statistical summary.
+
+"""
+
+import json
+import time
+from collections import defaultdict
+from functools import wraps
+from typing import TYPE_CHECKING, Any, Callable, DefaultDict, TypeVar
+
+from requests.models import Response
+from requests_toolbelt import MultipartEncoder
+
+if TYPE_CHECKING:
+    from .requests import Client
+
+_Callable = TypeVar("_Callable", bound=Callable[..., Response])
+
+
+class Profile:
+    """This is a class used to save statistical summary.
+
+    Attributes:
+        summary: The statistical summary.
+        running: Whether the statistical is running.
+
+    """
+
+    def __init__(self) -> None:
+        self.summary: DefaultDict[str, DefaultDict[str, Any]] = defaultdict(
+            lambda: defaultdict(int)
+        )
+        self.running = False
+
+    def synchronize_info(
+        self, key: str, response: Response, cost_time: float, data_length: int
+    ) -> None:
+        """Synchronize the giving information into summary.
+
+        Arguments:
+            key: the post method and url.
+            response: response of the post.
+            cost_time: cost time of the post.
+            data_length: the upload file length.
+
+        """
+        item = self.summary[key]
+        item["totalTime"] += cost_time
+        item["callNumber"] += 1
+        item["totalResponseLength"] += len(response.content)
+        item["totalFileSize"] += data_length
```

Comment:
why the key is "totalFileSize" but the variable name is `data_length`?
which is inconsistent